about.html

```diff
@@ -0,0 +1,130 @@
+                <li class="tes">My Page</li>
+                <li class="tes">Logout</li>
+            </ul>
+        </section>
+        <nav class="toolbar">
+            <a class="logo" href="#">
+                <img src="ressources/icons/logo.png" alt="">
+            </a>
+            <a class="button" href="#">
+                <img src="ressources/icons/mobile_menu 1.png" alt="Nav Button">
+            </a>
+            <ul class="cont-menu">
+                <ul class="menuDisplay">
+                    <li><a href="about.html" class="linkme" id="red-text">About</a></li>
+                    <li><a href="index.html" class="linkme">Home</a></li>
+                    <li class="border"><a href="#">CC Campaign</a></li>
+                </ul>
+            </ul>
+        </nav>
+        <section class="overlay"></section>
+        <div class="menu-container">
+            <button type="button" class="close-button"><img src="/ressources/icons/Icon - Cancel.png"></button>
+            <ul class="menuDisplay">
+                <li>
+                    <a href="#section1">
+                        <img src="ressources/icons/logo.png" alt="">
+                    </a>
+                </li>
+                <li><a href="about.html" class="linkme" id="red-text1">About</a></li>
+                <li><a href="index.html" class="linkme">Home</a></li>
+                <li class="border"><a href="#">CC Campaign</a></li>
+            </ul>
+        </div>
+        <section class="front">
+            <div class="cont-front">
+                <h2>ATTACK ON TITAN THE FINAL SEASON<br>PART 2</h2>
+                <hr class="about-line">
+                <p class="about-info-front">
+                    The second part of the final season of "The Final Season" is scheduled to air or soon
+                    of 2023. In the new trailer, Armin evokes the memories of three protagonist losistas when they were
+                    young, adding a layer of nostalgia to the plot. The phrase "Goodbye Ellen" spoken by Mikasa
+                    in the image key has generated speculation about the meaning suado. Fans expect twists
+                    surprises, shocking revelations and epic showdowns in this thrilling conclusion. He
+                    production team has worked hard to create a visually stunning experience and
+                    technically sound, while the writers have woven a cohesive and emotionally
+                    resonant. The second part " of The Final Season"ete promised to be a milestone in the history of
+                    anime and
+                    Fans are eager to describe how the final outcome will play out.
+                </p>
+                <p class="about-info-mail">If you want to contact the largest otaku community in Latin America</p>
+                <a href="#">
+                    <h4 class="about-mail">OtakuZone@gmail.com</h4>
+                </a>
+            </div>
+        </section>
```

Comment:
- [x] Kindly make the **Headline section** outside of the` header` tag to follow the best practices of HTML and make the `main` tag for all of your section